.travis.yml

```diff
@@ -60,7 +78,16 @@ install:
       pip install -U numpy pytest
     fi
   # install quilt from setup.py
-  - travis_retry pip install --only-binary 'tables' .
+  - travis_retry pip install --only-binary 'tables' ./compiler
+  - |
+    if [[ "$TEST_REGISTRY" = true ]]; then
+      travis_retry pip install -r registry/requirements.txt
+      pip install registry
+    fi
 
 script:
-  - pytest
+  - pytest compiler
+  - |
+    if [[ "$TEST_REGISTRY" = true ]]; then
+      pytest ./registry
+    fi
```

Comment:
LGTM - travis changes look good to me - they're passing of course, and I've reviewed the travis run log.

Post-merge, it would be nice to make it easier to run registry tests with less manual setup, e.g. move travis setup steps into a script that can be run locally as well (?)